wire-ios-request-strategy/Sources/Request Strategies/MLS/Actions/CountSelfMLSKeyPackagesActionHandler.swift

```diff
@@ -39,6 +39,7 @@ class CountSelfMLSKeyPackagesActionHandler: ActionHandler<CountSelfMLSKeyPackage
             return nil
         }
 
+        // TODO: .v8 requires ciphersuite or ciphersuites!
```

Comment:
Am I correct? https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/1593671681/API+changes+v7+v8

Reply:
Yes it looks like it. The optional query parameter was [added from v5](https://staging-nginz-https.zinfra.io/v5/api/swagger-ui/#/default/get_mls_key_packages_self__client__count) but we never used it. I think it's fine to start using if from v8. The question is... which cipher suite should we pass?